src/components/graphCard/graphCard.js

```diff
@@ -129,7 +130,7 @@ class GraphCard extends React.Component {
           };
           tempFiltered.stroke = chartColorGreenDark.value;
           tempFiltered.strokeDasharray = '4,3';
-          tempFiltered.strokeWidth = 2.5;
+          tempFiltered.strokeWidth = 3;
```

Comment:
Optional:
The history behind these changes is best described with the phrase "3rd time is a charm"

Since we're updating the Patternfly Charts base settings, and we're doing it again, very tempted to request you put a `ToDo:` annotation above this class method that says something along the lines of "[minimums are good, but we've changed this multiple times now. Styling for graphs/charts in the future, should be moved to the filters in the default props of the product views]" 

```
/**
   * ToDo: future consideration, maintain style minimums as default props in graphCard, move chartArea customizations to the filters used on product view configs.
   * FixMe: custom use of dash...
```

Reply:
I tried to figure out what you wanted me to do here, but I think I'm just not sure what the goal is.  Are you suggesting that we move the width values to props, so that we could change the widths in the future at the view level?  If so, I'm not sure why we would ever want to do that.  Shouldn't all charts of this type have the same widths?

If anything, I felt like we should be setting the widths deeper in, like in the ChartArea component.  I only put the changes here because that's where things were already being set.

Reply:
+1 to all same widths

Reply:
The goal was to consider it more of config based setup without specific settings being nested within a class method. There are settings that could be offset into the helpers, similar to the xAxis, yAxis, like "threshold" and "stacked". A chartArea helper could also be an answer, similar to your mention. 

We do at some point need to restructure the entire `chartArea.js` since there have been updates to the PF components that might, and I stress the might part, might help us cut code.

Reply:
We'll opening a follow up issue to help think about restructure.